.circleci/config.yml

```diff
@@ -69,10 +69,14 @@ commands:
             - "/home/circleci/go/pkg/mod/"
   notify-slack:
     description: Notify failures via Slack
+    parameters:
+      failures_only:
+        type: boolean
+        default: true
     steps:
       - run: exit 0
       - slack/status:
-          fail_only: false
+          fail_only: << parameters.failures_only >>
```

Comment:
out of curiosity, was there a reason for having the parameter naming different than the flag name we are setting?

Reply:
Not really, I though it could potentially be less problematic but it probably doesn't matter. I can make them the same if you think it's clearer.

Reply:
Actually since I have to rebase anyway I'll rename it :+1: